xtext-gradle-plugin/src/integTest/java/org/xtext/gradle/test/BuildingASimpleXtendProjectWithStandardCharsets.xtend

```diff
@@ -0,0 +1,56 @@
+package org.xtext.gradle.test
+
+import com.google.common.io.Files
+import java.nio.charset.Charset
+import java.util.Collection
+import org.gradle.api.tasks.compile.JavaCompile
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import org.junit.runners.Parameterized.Parameter
+import org.junit.runners.Parameterized.Parameters
+import org.xtext.gradle.tasks.XtextGenerate
+
+import static java.nio.charset.StandardCharsets.*
+
+@RunWith(Parameterized)
+class BuildingASimpleXtendProjectWithStandardCharsets extends AbstractIntegrationTest {
```

Comment:
I was undecided whether we need this test or not (or need to test all encodings). What do you think?